benchmark/single-source/ObjectiveCBridging.swift

```diff
@@ -79,6 +79,18 @@ public let ObjectiveCBridging = [
   BenchmarkInfo(name: "UnicodeStringFromCodable",
                  runFunction: run_UnicodeStringFromCodable, tags: ts,
                  setUpFunction: setup_UnicodeStringFromCodable),
+  BenchmarkInfo(name: "BridgedNSArrayObjectAtIndex",
```

Comment:
These names would read a little nicer with components separated by dots, as in `BridgedNSArray.objectAtIndex`, `BridgedNSArray.mutableCopyObjectAtIndex` etc.

Reply:
Sure, sounds good